DeleteTeamsWhereTeamMembersdontHaveGivenJobtitles/DeleteTeamsWhereTeamMembersdontHaveGivenJobtitles.ps1

```diff
@@ -0,0 +1,153 @@
+﻿#this script will Delete the all teams except given job titles match
+#Provide job title phrases separate by , 
+#for best practice run the script quote delete cmdlet
+
+$logfile = "C:\log_$(get-date -format `"yyyyMMdd_hhmmsstt`").txt"
```

Comment:
same comment for logging, consider Write-VerboseLog from template or other mechanism to have more verbose logging. Also, please do not use any absolute paths like c:\something in the script but use relative paths like .\mylogfile.txt only.